superset/initialization/__init__.py

```diff
@@ -701,3 +703,34 @@ class SupersetIndexView(IndexView):
     @expose("/")
     def index(self) -> FlaskResponse:
         return redirect("/superset/welcome/")
+
+    @staticmethod
+    def is_safe_url(target: str) -> bool:
```

Comment:
might belong in `superset/utils/urls.py` would be surprised if we don't already have a similar method there, grepping for `urlparse` or `next=` could point to a similar method.

Reply:
otherwise PR LGTM

Reply:
Thank you @mistercrunch ! I didn't find a similar method anywhere, so [4bb4916](https://github.com/apache/superset/pull/31692/commits/4bb49161c04f4cedeaeba30d9fabf9889fd9497c) creates `superset.utils.urls.is_safe_redirect_url(source_url, target_url)`.

Does this look ok now?